Gaijinizer patch/Map071.txt

```diff
@@ -123,39 +129,42 @@ processing. It is a little more on the surface
  果てには非人道的な人体実験に手を出してしまい、
  政府から危険人物指定を受け、一族からも追い出された。
 > CONTEXT: Map071/events/1/pages/0/70/Dialogue/yuuma
-However, excessive biological experiments,
-possession of dangerous drugs.
-In the end, he handed out an inhuman human
-experiment, The government designated him as a
-dangerous person and was expelled from his
-family.#MTLed
+However, there were biological experiments that
+went too far, and possession of dangerous drugs.
+In the end, she got involved in inhumane human
+experimentation, and the government designated her
+as a dangerous individual, and she was expelled
+from her clan.#Ron
 > END STRING
 
 > BEGIN STRING
 つまり\\C[2]『ヤバいやつ』\\C[0]だ。
 
  油断するなよ、\\N[1]。
 > CONTEXT: Map071/events/1/pages/0/75/Dialogue/yuuma
-In other words\\C[2] "Yaba"\\C[0] It is.
-Do not be off guard, \\N[1].#MTLed
+In other words,\\C[2] she's dangerous\\C[0].
+Stay on your guard, \\N[1].#Ron
 > END STRING
 
 > BEGIN STRING
 や、やだなぁ！人聞きん悪い……
 
 大丈夫よぉ、ワシは味方やけん……！！
 > CONTEXT: Map071/events/1/pages/0/86/Dialogue/other3
-Yay! It is bad people...All right, eagles are
-allies and ken.........! !#MTLed
+O- oh dear! How scandalous...
+
+It's okay, I'm on your side...!!#Ron
 > END STRING
 
 > BEGIN STRING
 ほ、ホラ！現にこうやって悪魔から助けたやろ！
 
 この転送装置を使って……
 > CONTEXT: Map071/events/1/pages/0/90/Dialogue/other3
-Hmmm! I actually helped from demon this way!
-With this transfer device ...#MTLed
+C- come on! I helped you escape that demon didn't
+I?
+
+Using this transfer device...#Ron
```

Comment:
We can line break here so we don't have to cut this off oddly...
>_**C-come on!
I helped you escape that demon, didn't I?**_